### PR TITLE
Tryday: upgrade projectors library to 7.0.0

### PR DESCRIPTION
**Pull request by tmarsteel**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
     <groupId>com.rebuy.archetypes</groupId>
     <artifactId>rebuy-silo-archetype</artifactId>
-    <version>9.9.1-SNAPSHOT</version>
+    <version>10.0.0-SNAPSHOT</version>
```
Comment (bjoernhaeuser) [inline pom.xml]: that should be done by the maven release plugin

Comment (tmarsteel) [reply pom.xml]: True.

**Comment by der-eismann** [comment]
Can I throw in some build-pipeline updates before you merge it?

**Comment by tmarsteel** [comment]
@der-eismann sure :)

**Comment by der-eismann** [comment]
Alright, I merged #113, you can continue now :slightly_smiling_face:

**Pull request merged**